library/std/src/sys/unix/process/process_vxworks.rs

```diff
@@ -196,6 +196,22 @@ impl ExitStatus {
     pub fn signal(&self) -> Option<i32> {
         if !self.exited() { Some(libc::WTERMSIG(self.0)) } else { None }
     }
+
+    pub fn core_dumped(&self) -> bool {
+        unimplemented!()
```

Comment:
It's probably better to just return false here, instead of causing a panic:
```suggestion
        false
```

Reply:
I changed the implementation to return `false` and added a comment explaining that the method is not yet fully implemented.